integration/js/README.md

```diff
@@ -142,3 +142,12 @@ Run the test:
 ```shell
 $KITE_HOME/scripts/mac/path/r configs/<test_name>.config.json
 ```
+Steps to add test file to Sauce Labs virtual machine
```

Comment:
[Not required now, can be done as a follow up or fixed later as it is integration tests related]:
First, thanks for adding this and finding a solution on using an external file for testing video. 
On the changes, can we make this a bit more clear for maintainers or contributors to follow more easily?

Few tips:
- Add hyper-links to places where this is currently done in tests or configurations.
- Suggest adding an explicit header and change the title to - "How to add a fake audio or video stream to your tests?" Current one renders "To create..." right after "Steps to add test file to Sauce Labs virtual machine". We would need a newline there. Adding a header should fix it.
- Can provide more context on "On sauce labs virtual machines files are stored in the below folder locations". Its a bit un-clear if we are to upload the files here using the SauceLabs storage APIs.